genesyscloud/telephony_providers_edges_site/resource_genesyscloud_telephony_providers_edges_site_test.go

```diff
@@ -49,15 +50,15 @@ func TestAccResourceSite(t *testing.T) {
 		t.Skipf("failed to delete location with number %s: %v", emergencyNumber, err)
 	}
 
-	location := gcloud.GenerateLocationResource(
+	location := location.GenerateLocationResource(
```

Comment:
Might be better to call this variable `locationResource` or something. At the moment it's the same as the imported package

Reply:
Same thing in a couple places below